<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>passwordgenerator</class>
 <widget class="QWidget" name="passwordgenerator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>297</width>
    <height>154</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>86</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>numbers / letters</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="PasswordBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>281</width>
     <height>61</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="alphaNumSpin">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>85</y>
     <width>48</width>
     <height>24</height>
    </rect>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>113</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Symbols</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="SymbolSpin">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>48</width>
     <height>24</height>
    </rect>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
  <widget class="QPushButton" name="refreshBtn">
   <property name="geometry">
    <rect>
     <x>183</x>
     <y>83</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CopyBtn">
   <property name="geometry">
    <rect>
     <x>184</x>
     <y>107</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
